src/server.ts

```diff
@@ -18,9 +19,10 @@ const build = (opts: FastifyServerOptions): FastifyInstance => {
   fastify.register(multipart);
   fastify.register(FastifySSEPlugin);
   fastify.register(cookie);
+  fastify.register(fastifyCompress);
   fastify.register(view, {
     engine: {
-      liquid: new Liquid({ extname: '.html', root: 'src/liquid', jekyllInclude: true, dynamicPartials: true }),
+      liquid: new Liquid({ extname: '.html', root: 'src/liquid', cache: true, jekyllInclude: true, dynamicPartials: true }),
```

Comment:
looks like it is recommended to enable this only in production environments. can we do that?
https://liquidjs.com/tutorials/use-in-expressjs.html#Caching

i personally like the dev experience of changing the view and having it reload without restarting the service